shell/renderer/renderer_client_base.cc

```diff
@@ -483,11 +515,78 @@ bool RendererClientBase::IsWebViewFrame(
 
   gin_helper::Dictionary frame_element_dict(isolate, frame_element);
 
-  v8::Local<v8::Object> internal;
-  if (!frame_element_dict.GetHidden("internal", &internal))
-    return false;
+  bool is_webview = false;
+  return frame_element_dict.GetHidden("isWebView", &is_webview) && is_webview;
+}
+
+void RendererClientBase::SetupMainWorldOverrides(
+    v8::Handle<v8::Context> context,
+    content::RenderFrame* render_frame) {
+  auto prefs = render_frame->GetBlinkPreferences();
+  // We only need to run the isolated bundle if webview is enabled
+  if (!prefs.webview_tag)
+    return;
+
+  // Setup window overrides in the main world context
+  // Wrap the bundle into a function that receives the isolatedApi as
+  // an argument.
+  auto* isolate = context->GetIsolate();
+
+  gin_helper::Dictionary isolated_api = gin::Dictionary::CreateEmpty(isolate);
+  isolated_api.SetMethod("allowGuestViewElementDefinition",
+                         AllowGuestViewElementDefinition);
```

Comment:
`&` should be placed before function pointers.

```suggestion
                         &AllowGuestViewElementDefinition);
```

Reply:
added